src/components/editor/Editor.jsx

```diff
@@ -4,182 +4,40 @@ import React, {Component} from 'react'
-                "valueDataType": {},
-                "defaults": []
-              }
-            },
-            {
-              "propertyLabel": "Carrier Type (RDA 3.3)",
-              "propertyURI": "http://id.loc.gov/ontologies/bibframe/carrier",
-              "repeatable": "true",
-              "resourceTemplates": [],
-              "type": "resource",
-              "valueConstraint": {
-                "valueTemplateRefs": [],
-                "useValuesFrom": [
-                  "http://id.loc.gov/vocabulary/carriers"
-                ],
-                "valueDataType": {
-                  "dataTypeURI": "http://id.loc.gov/ontologies/bibframe/Carrier"
-                },
-                "repeatable": "true",
-                "editable": "false",
-                "defaults": [
-                  {
-                    "defaultURI": "http://id.loc.gov/vocabulary/carriers/nc",
-                    "defaultLiteral": "volume"
-                  }
-                ]
-              },
-              "mandatory": "false",
-              "remark": "http://access.rdatoolkit.org/3.3.html"
-            },
-            {
-              "propertyLabel": "WITH ALL VALUE CONSTRAINTS",
-              "propertyURI": "http://id.loc.gov/ontologies/fake",
-              "repeatable": "true",
-              "resourceTemplates": [],
-              "type": "resource",
-              "valueConstraint": {
-                "defaults": [
-                  {
-                    "defaultLiteral": "DEFAULT",
-                    "defaultURI": "http://default"
-                  }
-                ],
-                "editable": "true",
-                "languageLabel": "LANGUAGE LABEL",
-                "languageURI": "http://id.loc.gov/vocabulary/languages/eng",
-                "remark": "REMARK",
-                "repeatable": "false",
-                "useValuesFrom": [
-                  "http://VALUES"
-                ],
-                "validatePattern": "PATTERN",
-                "valueDataType": {
-                  "dataTypeLabel": "Classification item number",
-                  "dataTypeLabelHint": "HINT",
-                  "dataTypeURI": "http://id.loc.gov/ontologies/bibframe/itemPortion",
-                  "remark": "REMARK"
-                },
-                "valueLanguage": "VALUE LANGUAGE",
-                "valueTemplateRefs": [
-                  "profile:bf2:Identifiers:Barcode"
-                ]
-              },
-              "mandatory": "false",
-              "remark": "http://access.rdatoolkit.org/3.3.html"
-            }
-          ]
-        }
-      ]
+
+    this.getResourceTemplate = this.getResourceTemplate.bind(this)
+
+    // TODO: temporarily hardcoded here.
+    //  Selecting a resource template will happen in the left-nav "Starting Points" menu,
+    //   another child of the Editor component;  it will call this.getResourceTemplate
+    const defaultResourceTemplate = 'resourceTemplate:bf2:Monograph:Instance'
```

Comment:
You and @SaravShah will probably want to use a resource template that has more than one ` "type": "literal"` so should this be changed to (e.g.) `resourceTemplate:bf2:WorkTitle` or will you just change it on you branch when you get to that? Soon we will need to hook up the StartingPoints for this...